### ISSUE TITLE
keras callback not defined

### ISSUE THREAD

**Issue by ksaidin** [Nipun-1997/SentenceCorrection#1]
Hello, thank you for sharing your codes.

File: Sequence2SequnceWithGRULSTM.ipynb

model.fit([encoder_input_data, decoder_input_data],decoder_output_data,validation_data=([cv_encoder_input_data,cv_decoder_input_data],cv_decoder_output_data),batch_size=64,epochs=260,callbacks=[rl])

I couldn't find where "rl" is defined.

**Comment by Nipun-1997** [comment]
yes.. i have gone through the code and rl has not been used. Please remove callback before running the code

**Comment by ksaidin** [comment]
I doubt callback is not been used. 

It says 

> Epoch 00241: ReduceLROnPlateau reducing learning rate to 1.3421771291177721e-05. 

in between training logs.

I think there should be something like:

> rl = ReduceLROnPlateau(monitor='val_loss', factor=0.2, patience=5, min_lr=0.001)

**Comment by Nipun-1997** [comment]
yes i gone through it and that piece of code is missing from the notebook
